{{cookiecutter.project_name}}/Dockerfile

```diff
@@ -1,12 +1,14 @@
 FROM python:3.6
 
+WORKDIR /app
+
 ENV DJANGO_SETTINGS_MODULE {{cookiecutter.project_name}}.settings
 ENV UWSGI_INI /app/uwsgi.ini
 
-COPY . /app
-WORKDIR /app
+COPY requirements.txt /app
+RUN pip install -U -r requirements.txt
```

Comment:
`--upgrade --requirement` would make this much more readable (like in the Makefile).

Do we really need to `-U`? Shouldn't the requirement file contain the exact version and be enforced anyhow? Do we really want/need to replace what is already in the upstream container?

Reply:
Indeed long version are better. Adjusted.

In case of the --upgrade flag. Usually it doesn't change anything. The only dep which could cause issue I see is `pip`. And someone for some reason (e.g. a another package can only be installed with a newer pip version) could indeed try to stick to a newer pip version and we should allow this.